bootstrap.sh

```diff
@@ -0,0 +1,16 @@
+apt-get update # needed to install python-software-properties
+apt-get install -y python-software-properties # needed to add apt-get repos
+apt-add-repository -y ppa:brightbox/ruby-ng-experimental # needed to install ruby2.0
+apt-get update # recognize new repo
+
+# utils

```

Comment:
Yes, they're necessary because this is Precise64 which is Ubuntu 12. apt-get repos for a version this old only point to Ruby 1.9.3 at most.